app/models/user.rb

```diff
@@ -9,4 +9,13 @@ class User < ApplicationRecord
   def password_required?
     !persisted? || password.present? || password_confirmation.present?
   end
+
+  def member_in_sections
```

Comment:
FYI this hackiness won't be required when we move to a collaborator model and pundit.